### PR TITLE
Fix locations

### PR DESCRIPTION
**Pull request by baikauskaite**
- Added `LocationIds` to Product and Service
- Fixed types of leftover foreign keys
- Removed `ServiceOrder`

Ignoruokime force pushą

### CODE REVIEW

File: PoSSapi/Classes/Product.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@ public class Product
     public string Name { get; set; }
     public decimal Price { get; set; }
     public decimal Tax { get; set; }
-    public Discount Discount { get; set; }
-    public Category Category { get; set; }
+    public string DiscountId { get; set; }
```
Comment (Arnasst) [inline PoSSapi/Classes/Product.cs:9]: Discount class already has a property TargetId and DiscountTargetType(category, service, product). I wonder if having this property then has any value? If you agree that it doesn't, then let's just comment it on Confluence.

File: PoSSapi/Classes/Service.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@ public class Service
     public string Name { get; set; }
     public string? EmployeeId { get; set; }
     public decimal Price { get; set; }
-    public Object Duration { get; set; } //Not specified in the diagram
-    public Category Category { get; set; }
+    public int Duration { get; set; } // in minutes
```
Comment (Arnasst) [inline PoSSapi/Classes/Service.cs:9]: Perhaps store time in seconds?

**Review by Arnasst — COMMENTED** [review]
image

File: PoSSapi/Classes/Product.cs (hunk 1 above)
Comment (baikauskaite) [reply PoSSapi/Classes/Product.cs:9]: I agree that it doesn't have much value. I'll document our decision to take it out.

File: PoSSapi/Classes/Service.cs (hunk 2 above)
Comment (baikauskaite) [reply PoSSapi/Classes/Service.cs:9]: :+1:

**Review by Arnasst — APPROVED** [review]
Looks great!

**Pull request merged**